functions/go/starlark/third_party/sigs.k8s.io/kustomize/kyaml/fn/runtime/starlark/context.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 The Kubernetes Authors.
```

Comment:
it will be useful to bookkeep which version we are importing in a README.md in `third_party/README.md` and it will be better to import a tagged version if possible.

Reply:
Done